StripeFinancialConnections/StripeFinancialConnections/Source/API Bindings/FinancialConnectionsAPIClient.swift

```diff
@@ -480,7 +484,86 @@ extension STPAPIClient: FinancialConnectionsAPIClient {
 
     // MARK: - Networking
 
-    func saveAccountsToLink(
+    func saveAccountsToNetworkAndLink(
```

Comment:
this function is based off [the following web code](https://git.corp.stripe.com/stripe-internal/stripe-js-v3/blob/master/src/linkedAccounts/inner/hooks/v3/useSaveToNetworkAndLink.ts#L101-L133)